opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.hpp

```diff
@@ -91,16 +102,26 @@ namespace Opm {
 
         double getRegionMultiplier(size_t globalCellIdx1, size_t globalCellIdx2, FaceDir::DirEnum faceDir) const;
 
-        std::array<size_t,3> getSize() const;
-        const std::vector<MULTREGTRecord>& getRecords() const;
-        ExternalSearchMap getSearchMap() const;
-        const std::map<std::string, std::vector<int>>& getRegions() const;
-        const std::string& getDefaultRegion() const;
-
         bool operator==(const MULTREGTScanner& data) const;
         MULTREGTScanner& operator=(const MULTREGTScanner& data);
 
+        template<class Serializer>
+        void serializeOp(Serializer& serializer)
+        {
+            serializer(nx);
+            serializer(ny);
+            serializer(nz);
+            serializer.vector(m_records);
+            ExternalSearchMap searchMap = getSearchMap();
```

Comment:
Isn't this function removed above?

Reply:
well, yes, and readded as private below.